Web/View/Channels/Show.hs

```diff
@@ -43,14 +44,20 @@ instance View ShowView where
 
                     bodyPlaceholder = "Send message to #" <> get #name channel
 
-renderMessage :: Message -> Html
-renderMessage message = [hsx|
+getUserEmail :: Id' "users" -> [User] -> Text
```

Comment:
Typically in IHP apps it's written like this:

```suggestion
getUserEmail :: Id User -> [User] -> Text
```

Reply:
Thanks for the suggestion!